src/rpc/handlers/NFTInfo.cpp

```diff
@@ -99,37 +23,28 @@ doNFTInfo(Context const& context)
         return *status;
     auto const lgrInfo = std::get<ripple::LedgerInfo>(maybeLedgerInfo);
 
-    std::optional<Backend::NFT> dbResponse =
+    std::optional<Backend::NFT const> const dbResponse =
         context.backend->fetchNFT(tokenID, lgrInfo.seq, context.yield);
     if (!dbResponse)
         return Status{Error::rpcOBJECT_NOT_FOUND, "NFT not found"};
 
-    response["nft_id"] = ripple::strHex(dbResponse->tokenID);
-    response["ledger_index"] = dbResponse->ledgerSequence;
-    response["owner"] = ripple::toBase58(dbResponse->owner);
+    response[JS(nft_id)] = ripple::strHex(dbResponse->tokenID);
+    response[JS(ledger_index)] = dbResponse->ledgerSequence;
+    response[JS(owner)] = ripple::toBase58(dbResponse->owner);
     response["is_burned"] = dbResponse->isBurned;
-
-    response["flags"] = ripple::nft::getFlags(dbResponse->tokenID);
-    response["transfer_fee"] = ripple::nft::getTransferFee(dbResponse->tokenID);
-    response["issuer"] =
+    if (dbResponse->uri)
+        response[JS(uri)] = ripple::strHex(dbResponse->uri.value());
+    else
+        response[JS(uri)] = nullptr;
+
+    response[JS(flags)] = ripple::nft::getFlags(dbResponse->tokenID);
+    response["transfer_rate"] =
+        ripple::nft::getTransferFee(dbResponse->tokenID);
+    response[JS(issuer)] =
         ripple::toBase58(ripple::nft::getIssuer(dbResponse->tokenID));
     response["nft_taxon"] =
         ripple::nft::toUInt32(ripple::nft::getTaxon(dbResponse->tokenID));
-    response["nft_sequence"] = ripple::nft::getSerial(dbResponse->tokenID);
-
-    if (!dbResponse->isBurned)
-    {
-        auto const maybeURI = getURI(*dbResponse, context);
-        // An error occurred
-        if (Status const* status = std::get_if<Status>(&maybeURI); status)
-            return *status;
-        // A URI was found
-        if (std::string const* uri = std::get_if<std::string>(&maybeURI); uri)
-            response["uri"] = *uri;
-        // A URI was not found, explicitly set to null
-        else
-            response["uri"] = nullptr;
-    }
+    response[JS(nft_serial)] = ripple::nft::getSerial(dbResponse->tokenID);
```

Comment:
is this a field name change from before?

Reply:
yes, to match what we have in `account_nfts`. see PR description

Reply:
I think this is a no-go at this point, nft_info is already live and being used by numerous people, lets not break their script.

Reply:
You could add both for now and later we can have a process to formally remove the former over a period of releases with deprecation warnings in changelog etc.

Reply:
spoke about this w alex. because there's already a breaking change in the API wrt the `uri` field (it's no longer hex-decoded to resolve an unrelated issue) we think we should just make these other API changes while we're here anyway.